api.php

```diff
@@ -1072,6 +1054,25 @@ public function run($userID, $permissionIsExplicit) {
 			$where = "toCountryID = 0";
 		}
 
+		// Press Types
+		// Regular - Global and Private messaging allowed.
+		// PublicPressOnly - Only Global messaging allowed.
+		// NoPress - No messaging allowed.
+		// RulebookPress - No messaging allowed during 'Builds' and 'Retreats' phases.
+		else if ($countryID != $toCountryID && $pressType == 'NoPress' || $countryID != $toCountryID && $pressType == 'RulebookPress' && $gamePhase == 'Builds' || $countryID != $toCountryID && $pressType == 'RulebookPress' && $gamePhase == 'Retreats') {
```

Comment:
Can you group these operator statements with parentheses? Might be a bit difficult to understand what is happening here, assuming the operator precedence is all good. 

`$countryID != $toCountryID && $pressType == 'NoPress' || $countryID != $toCountryID && $pressType == 'RulebookPress' && $gamePhase == 'Builds'` could be `($countryID != $toCountryID && $pressType == 'NoPress') || ($countryID != $toCountryID && $pressType == 'RulebookPress' && $gamePhase == 'Builds')`

Reply:
@bryancodazen  I agree it is hard to read. I added parentheses and separated the conditional checks onto their own line hopefully this is easier to understand.